spec/models/suite_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Suite, type: :model do
+  describe 'basic validations' do
+    it_behaves_like 'does not allow nil', :number
```

Comment:
Is there a reason why you decided to build your own validation checkers as opposed to using `shoulda_matchers`? I feel like they cover all of these test cases.